src/Client.php

```diff
@@ -49,143 +49,163 @@ class Client implements OpenShiftClientInterface {
    * @var array
    */
   protected $resourceMap = [
-    'secret' => [
+    'secret'                => [
       'create' => [
         'action' => 'POST',
-        'uri' => '/api/v1/namespaces/{namespace}/secrets',
+        'uri'    => '/api/v1/namespaces/{namespace}/secrets',
```

Comment:
Lines up nicely in the editor. Meh. I'd prefer to leave it in as it looks a little nicer.